addons/web_editor/static/src/js/wysiwyg/widgets/link_dialog.js

```diff
@@ -48,6 +48,7 @@ var LinkDialog = Dialog.extend({
             {type: this.isButton ? 'link' : '', label: _t("Link"), btnPreview: 'link'},
             {type: 'primary', label: _t("Primary"), btnPreview: 'primary'},
             {type: 'secondary', label: _t("Secondary"), btnPreview: 'secondary'},
+            {type: 'success', label: _t("Success"), btnPreview: 'success'},
```

Comment:
@Guillaume-gdi We don't want to suggest "success" as a new button type, that's actually something we have been wanting to discourage in the past and try to make users only use "primary", "secondary" or "link" buttons (even though we added the "custom" button possibility in 15.0, we still show "primary" etc as first suggestions).